include/fmt/format-inl.h

```diff
@@ -2594,9 +2594,8 @@ extern "C" __declspec(dllimport) int __stdcall WriteConsoleW(  //
 }  // namespace detail
 #endif
 
-FMT_FUNC void vprint(std::FILE* f, string_view format_str, format_args args) {
-  memory_buffer buffer;
-  detail::vformat_to(buffer, format_str, args);
+namespace detail {
+FMT_FUNC void print_buffer(std::FILE* f, const memory_buffer& buffer) {
```

Comment:
I think it would be slightly better to pass the text as a `string_view` instead of a buffer. This also means that the name would have to change. Maybe `detail::print`?

Reply:
Done, the most challenging part of this change was to name the new argument correctly 😄, but since you mentioned that it's the text, I used `text` as a name.